ROADMAP.md

```diff
@@ -2,7 +2,7 @@
 
 Each quarter the team will highlight areas of focus for our work and upcoming research.
  
-We select items for inclusion in the roadmap from the Top 10 Community Issues, [core services](docs/CORE_SERVICES.md), and internal priorities. When community pull requests exist for a given item, we will prioritize working with the original authors to include their contributions. If the author can no longer take on the implementation, HashiCorp will complete any additional work needed. 
+We select items for inclusion in the roadmap from the Top 10 Community Issues, [core services](info/CORE_SERVICES.md), and internal priorities. When community pull requests exist for a given item, we will prioritize working with the original authors to include their contributions. If the author can no longer take on the implementation, HashiCorp will complete any additional work needed. 
```

Comment:
Please note that the issue preventing non provider documentation files from living in `docs/` was fixed and now this file lives there again via: #13227

Reply:
Fixed.